examples/channels/multi/multi.C

```diff
@@ -0,0 +1,51 @@
+#include "multi.h"
+#include <cstdlib>
+#include <ctime>
+#include <numeric>
+
+Main::Main(CkArgMsg *m) {
```

Comment:
Either make this [nokeep] or delete the msg, same for the other example.

Reply:
I didn't know `nokeep` was an option for the main entry point, I'll go ahead and do that.